java-rx-smgrsdk/src/main/java/io/vantiq/utils/VertxWideData.java

```diff
@@ -88,9 +88,8 @@ public static <K, V> Cache<K, V> getVertxWideCache(Vertx vertx, String mapName,
      */
     public static <T> T getVertxWideInstance(Vertx vertx, String instanceName, Supplier<T> instanceSupplier) {
         LocalMap<String, VertxWideData<T>> globalsMap = vertx.sharedData().getLocalMap(GLOBALS_MAP_NAME);
-        VertxWideData<T> instanceGlobal = globalsMap.computeIfAbsent(instanceName, key ->
-            new VertxWideData<>(instanceSupplier.get())
-        );
+        globalsMap.putIfAbsent(instanceName, new VertxWideData<>(instanceSupplier.get()));
```

Comment:
the Map returned from io.vertx.rxjava3.core.Vertx does not have a `computeIfAbsent`. why? no idea. using it instead of core.Vertx has a handful of benefits, and some downsides...